src/store/cart-store.js

```diff
@@ -41,6 +31,36 @@ class CartStore extends Store {
 
 const store = new CartStore();
 
+function startTimer(period) {
+    timer = setTimeout(() => {
+        if(store.getIsLongterm()){
+            if(StudentController.isValidLongtermData(store.getDueDate(), store.getProfessor())){
```

Comment:
Negative, that would cause the normal checkout to run which i DONT want to happen. If the longterm data is invalid then nothing should happen since error modals will pop up.